frontend/src/components/dashboard/demo/create-demo-modal/CreateDemoModal.tsx

```diff
@@ -12,6 +12,9 @@ import { ModalContext } from 'contexts/modal.context'
 import { useHistory } from 'react-router-dom'
 import { createCampaign } from 'services/campaign.service'
 import DemoVideoModal from 'components/dashboard/demo/demo-video-modal'
+
+import type { Campaign } from 'classes/Campaign'
```

Comment:
Out of curiosity, what's the difference between `import` and `import type`?

Reply:
I'm not fully clear on the specifics of this, but what I understand is:

`import type` only imports the declarations for type checking, but doesn't introduce a runtime dependency on what was imported. So, after transpilation, whatever was imported via `import type` shouldn't appear in the code.

There's a short doc on TypeScript's website on it as well: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html

Reply:
Makes sense. Guess it should have _some_ effect on the bundle size. Was originally concerned with the following from happening:

```ts
import type { Campaign } from 'classes/Campaign'
...
const campaign = new Campaign(...)
```

But the compiler will catch this and recognize that only the type was imported.